src/connectMultireducer.js

```diff
@@ -28,22 +30,15 @@ export default function connectMultireducer(mapStateToProps, actions = {}) {
       generateConnectedComponent({multireducerKey}) {
         this.ConnectedComponent =
           connect(
-            (state, ownProps) => {
-              const multireducerKeys = Object.keys(state.multireducer);
-              if (!multireducerKeys.filter(key => key === multireducerKey)[0]) {
-                throw new Error(`No state for multireducer key "${multireducerKey}". You initialized multireducer with "${multireducerKeys.join(', ')}".`);
-              }
-              const slice = state.multireducer[multireducerKey];
-              return mapStateToProps ? mapStateToProps(slice, ownProps) : slice;
-            },
-            multireducerBind(actions, multireducerKey)
+            wrapMapStateToProps(mapStateToProps, multireducerKey),
+            wrapMapDispatchToProps(mapDispatchToProps, multireducerKey),
+            ...rest
```

Comment:
Or am I expected, as the user of the library, to slice my own state: `state.multireducer[key]` in `mapStateToProps`?